openfecli/commands/gather.py

```diff
@@ -64,7 +121,9 @@ def get_names(result) -> tuple[str, str]:
         return toks[0], toks[2]
 
 
-def get_type(res):
+def get_type(res:dict)->Literal['vacuum','solvent','complex']:
+    """Determine the simulation type based on the component types."""
```

Comment:
Maybe here it's worth being a bit explicit, it's determining the simulation type based on the key indices of the components.

The reason for being clearer here is that this is super super flaky / bad practice! We don't enforce key names in the RFE protocol so this won't work if you use the Python API to set up your transformations and then decided to use gather.

I didn't realise this even happened.. we should probably raise an issue about it.